<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmInstallationDialog</class>
 <widget class="QDialog" name="ConfirmInstallationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm Installation Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="thirdPartyWarningLabel">
     <property name="text">
      <string>&lt;p&gt;&lt;font color=&quot;orange&quot;&gt;WARNING:&lt;/font&gt; Proceeding will enable a third-party apt repository. A compromised apt repository can install malicious software in place of any package on your system during normal software updates. Only proceed if you trust this application's developers and their security practices. See &lt;a href=&quot;https://www.kicksecure.com/wiki/Install_Software#Third_Party_Repository_Warning&quot;&gt;Third Party Repository Warning&lt;/a&gt; for more information.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="actionInfoLabel">
     <property name="text">
      <string>actionInfoLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="commandLabel">
     <property name="text">
      <string>&lt;code&gt;commandLabel&lt;/code&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Click 'Continue' to proceed with these changes, or 'Back' to return to the 'Choose Installation Options' page.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset theme="go-previous">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue</string>
       </property>
       <property name="icon">
        <iconset theme="go-next">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
